scripts/uosc/elements/Menu.lua

```diff
@@ -237,13 +237,14 @@ function Menu:update(data)
 	end
 	-- Apply search suggestions
 	for _, menu in ipairs(new_menus) do
-		if menu.search_suggestion then
+		if menu.search_suggestion then menu.search.query = menu.search_suggestion end
```

Comment:
This is a new menu that didn't have search initialized yet, so `menu.search` will be `nil` here, no?

Reply:
Doesn't the loop above that take care of it? Otherwise the same problem exists on the main branch.

Reply:
True about the above loop 😐.